Do not log the latest version if it's unknown (#1967)
FerretDB/FerretDB: website/docs/telemetry.md

## Version notification

When a FerretDB update is available, the telemetry service sends periodic reports containing information about the latest FerretDB version.

> This information is logged in the server logs, `startupWarnings`, and `serverStatus` command output.

Author: `serverStatus` never logged version update information, only the current telemetry state

While you may not upgrade to the latest release immediately,
ensure that you update early to take advantage of recent bug fixes, new features, and performance improvements.

## Configure telemetry

The telemetry service has three state settings: `enabled`, `disabled`, and `undecided` (default).

> The latter acts as if the telemetry reporter is `enabled` with two differences:

Reviewer: What "the latter" refers to? If it's about `undecided`, let's say `undecided` for clarity.

Author: I'm not sure if that would improve clarity (what else "the latter" could mean?), but that would definitely increase tautology by using the same word next to each other

* When `enabled`, the first report is sent right after FerretDB starts.
  If `undecided`, the first report is delayed by one hour.
  That should give you enough time to disable it if you decide to do so.

> * Similarly, when `enabled`, the last report is sent right before FerretDB shutdowns.

Reviewer 2: ```suggestion
* Similarly, when `enabled`, the last report is sent right before FerretDB shuts down.
```

  That does not happen when `undecided`.

:::info

> `undecided` state does not change itself into `enabled` after the first or any other report.
> Only explicit user action (see below) could change it to `enabled` or `disabled`.

Reviewer 2: ```suggestion
`undecided` state does not automatically change into `enabled` or `disabled` after the first or any other report.
Explicit user action is required (see below) to change an `undecided` state to `enabled` or `disabled`.
```

When the state setting is `undecided`, users have a delay period of one hour after startup before telemetry data is reported.
`undecided` state does not change itself into `enabled` after the first or any other report.
Only explicit user action (see below) could change it to `enabled` or `disabled`.
:::

### Disable telemetry

> We urge you not to disable this service, as its insights will help us enhance our software.

Author: Users can't disable _service_ (beacon.ferretdb.io)
